nixos/modules/virtualisation/grow-partition.nix

```diff
@@ -24,7 +24,12 @@ with lib;
       copy_bin_and_libs ${pkgs.gnused}/bin/sed
       copy_bin_and_libs ${pkgs.utillinux}/sbin/sfdisk
       copy_bin_and_libs ${pkgs.utillinux}/sbin/lsblk
-      cp -v ${pkgs.cloud-utils}/bin/.growpart-wrapped $out/bin/growpart
+
+      substitute "${pkgs.cloud-utils}/bin/.growpart-wrapped" "$out/bin/growpart" \
+        --replace "${pkgs.bash}/bin/sh" "/bin/sh" \
+        --replace "awk" "gawk" \
+        --replace "sed" "gnused"
```

Comment:
This looks strange: you copy `bin/sed` and then replace sed with gnused. And why don't we copy `bin/awk`?

Reply:
I guess this is a name collision with busybox.

Reply:
I actually only restored the previous behavior. Do we have any name collision with gawk and busybox?

Reply:
Super super weird to modify a store path in place taking advantage of the fact that you're in the initrd 😱

Reply:
Never mind, it's not in place, but still weird to patch the wrapper when we own it.